### PR TITLE
Updated ui-tests Configuration in Contributing.md

### PR DESCRIPTION
**Pull request by jayeshsingh9767**
While setting up UI tests on my Local System, I have faced issue while using jlpm, Got to know from @krassowski  that we need to install Packages of jlpm inside folder `ui-tests`. 

So, This PR will update the [CONTRIBUTING.md](https://github.com/jupyter/notebook/blob/main/CONTRIBUTING.md) so that future contributors doesn't face the same.

### CODE REVIEW

**Review by krassowski — APPROVED** [review]
Thank you!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -79,6 +79,12 @@ There are also end to end tests to cover higher level user interactions, located
 
 ```bash
 cd ui-tests
+#install required packages for jlpm
+jlpm
+
+#install playwright
+npx playwright install
```
Comment (jtpio) [inline CONTRIBUTING.md:86]: Maybe we can stick to `jlpm` for consistency here?

```suggestion
jlpm playwright install
```

Comment (jayeshsingh9767) [reply CONTRIBUTING.md:86]: Thanks @jtpio for pointing out.

**Review by jtpio — APPROVED** [review]
Thanks!

**Pull request merged**